addon/services/unified-event-handler.js

```diff
@@ -40,8 +40,11 @@ const generateId = (function() {
 }());
 
 export default Ember.Service.extend(Ember.Evented, {
-  // Keeps track of the handlers that have been registered
-  [_HANDLER_MAP]: {},
+  init() {
+    this._super(...arguments);
+
+    this[_HANDLER_MAP] = {};
```

Comment:
Let's update this to `Object.create(null)`. We really just want an empty dictionary.